src/sass/_permission_identities.scss

```diff
@@ -0,0 +1,63 @@
+.permission-identities,
+.permission-identity-groups {
+  .search-wrapper {
+    margin-bottom: -0.3rem; // needed to align button with search filter
+    width: 100%;
+  }
+
+  @include mobile {
+    .name-link {
+      .p-chip {
+        display: none;
+      }
+    }
+  }
+}
+
```

Comment:
```suggestion
```
This seems to be unused?

Reply:
the entire file can be removed actually.